locales/locales.php

```diff
@@ -972,6 +1010,8 @@ public function __construct() {
 		$gsw->country_code = 'ch';
 		$gsw->wp_locale = 'gsw';
 		$gsw->slug = 'gsw';
+		$gsw->variant_root = $de->slug;
```

Comment:
That would be handled somewhere else though 🙂 `de_DE_formal` is not listed in `locales.php` after all, it's probably just a WordPress.org thing.